orchagent/neighorch.cpp

```diff
@@ -137,16 +137,24 @@ void NeighOrch::doTask(Consumer &consumer)
         }
 
         string alias = key.substr(0, found);
-        Port p;
 
-        if (!gPortsOrch->getPort(alias, p))
+        if (alias == "eth0" || alias == "lo" || alias == "docker0")
         {
             it = consumer.m_toSync.erase(it);
             continue;
         }
 
+        Port p;
+        if (!gPortsOrch->getPort(alias, p))
+        {
+            SWSS_LOG_INFO("Port %s doesn't exist", alias.c_str());
+            it++;
+            continue;
```

Comment:
can we ask the neighsyncd to periodically send the neighbor information to app db. 

this solution creates concerns that such neighbor information may never gets deleted from app db in case the port name is wrong.